src/Compiler/Microsoft.AspNetCore.Razor.Language/legacyTest/Legacy/ImplicitExpressionEditHandlerTest.cs

```diff
@@ -446,7 +446,7 @@ public void IsAcceptableInsertionInBalancedParenthesis_BalancedParenthesis_Retur
     private static Syntax.MarkupTextLiteralSyntax GetSyntaxNode(SourceLocation start, string content)
     {
         var builder = SyntaxListBuilder<SyntaxToken>.Create();
-        var tokens = CSharpLanguageCharacteristics.Instance.TokenizeString(content).ToArray();
+        var tokens = LegacyCSharpLanguageCharacteristics.Instance.TokenizeString(content).ToArray();
```

Comment:
This is probably more of a Nit than a must-fix, but I do wonder if we might consider a different prefix than `Legacy`. I actually think its a good name for it, but the rest of the codebase has a whole bunch of stuff called `LegacyX` which refers to CSHtml vs Components. 

Ideally this won't be around long, but I am well aware of how often that statement proves to be wrong, so would be nice if we could differentiate the 'old' parser from the 'legacy' cshtml only stuff.

Reply:
Now, as to what we could call it? Original? Old? Previous? 

I personally quite like 'Native' as it has the same idea of the native/roslyn codebase, and while it's not native code, it was 'natively implemented' but it's probably even more confusing. Yes I am overthinking this, but we *need* to know what color the bikeshed is going to be :)